aten/src/ATen/core/ivalue.cpp

```diff
@@ -932,6 +932,34 @@ getClassConverter() {
   return classConverter;
 }
 
+// Needs to be in this .cpp file to access the full definition of PyObjectHolder
+std::vector<std::reference_wrapper<const at::DataPtr>> ivalue::Future::extractDataPtrs(
```

Comment:
assuming no change in the implementation, except this is moved from `CUDAFuture`.

Reply:
Yes I just copy-pasted.